src/pages/experiments/[experimentId]/data-processing/data-integration/components/CalculationConfig.jsx

```diff
@@ -19,11 +20,22 @@ import {
   QuestionCircleOutlined,
 } from '@ant-design/icons';
 
+import SeuratV4Options from './SeuratV4Options';
```

Comment:
why don't we rename this file to `MethodOptions` to match the name of the component?

Reply:
It should be the other way around,

Reply:
makes sense!

Reply:
changed in new commit